rpmlint/checks/FileDigestCheck.py

```diff
@@ -49,7 +52,12 @@ def _get_digest_configuration_group(self, pkgfile):
             for location in locations:
                 try:
                     if path.relative_to(location):
-                        return group
+                        if not self.include_only_in_group[group]:
+                            return group
+                        else:
+                            for glob in self.include_only_in_group[group]:
+                                if fnmatch(path, glob):
```

Comment:
should we maybe only match against the basename of the `path`? Otherwise false positives could occur. Say we have an `IncludeOnly="*lib*.so"` and encounter a path like `/usr/lib/security/plugin.so` then it would still match against the complete path, while the basename would not match.